content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Sensitive Data Masking"
+description: "How to manage sensitive data masking"
+tags: "admin policy management sensitive data masking"
+-->
+
+Take advantage of Contrast's data masking feature to limit risk to your organization and meet compliance requirements.  
+
+## How It Works
+
+Contrast's data masking feature protects sensitive data in your applications by preventing the Contrast agent from sending it to external systems: the Contrast UI, syslog or the security log. Data masking can also protect information by preventing the agent from sending it as part of Contrast vulnerability or attack reports. 
+
+Sensitive data masking is part of policy management by default for organizations with at least one active Assess or Protect license. Contrast offers several categories of sensitive data, or data types, that are comprised of specific keywords that the agent automatically identifies and redacts in reports. 
+
+### Agent identification and masking
+
+Contrast agents mask sensitive data in query parameters, request headers, cookies and forms. Your agent identifies sensitive data by searching for specific keywords used in the input name. If the agent finds a match, it redacts the value for that input, and replaces it with a placeholder with the format `contrast-redacted-{datatype}`, where `datatype` is the category of sensitive data to which the keyword belongs.
+
+Contrast agents do **not** mask individual fields in request bodies with a content type other than `application/x-www-form-urlencoded`; however, you can configure the agent to mask the entire request body. Contrast agents also do not mask data that appears in the data flow portion of a vulnerability report, if using Assess, or in the vector of an attack event, if using Protect. 
+
+> **Note:** Contrast agents make a “best effort” attempt to avoid printing sensitive data in debug log statements; however, it’s possible that sensitive data could appear in the Contrast log, if the log level is set to DEBUG or lower. Whenever possible, you should avoid setting production systems to log at DEBUG or lower. If a system that deals with sensitive data is set to log at DEBUG or lower, you should take steps to ensure that those logs are not being sent to an external system to avoid leaking any sensitive data. 
```

Comment:
> Contrast agents make a “best effort” attempt to avoid printing sensitive data in debug log statements; however, it’s possible that sensitive data could appear in the Contrast log, if the log level is set to DEBUG or lower.

I think this works a little better if we refer to this specific log as just the "Contrast log" or `contrast.log` instead of calling it debug log statements. That latter conflates log level with the actual log and lacks specificity. I think it's consistent with how we refer to this log based on https://docs.contrastsecurity.com/installation-setupconfig.html#log